pilot/pkg/config/clusterregistry/clusterregistry.gotmpl

```diff
@@ -0,0 +1,16 @@
+apiVersion: clusterregistry.k8s.io/v1alpha1
+kind: {{.Kind}}
+metadata:
+  name: {{.Name}}
+  annotations:
+    config.istio.io/pilotEndpoint: "{{.PilotIP}}:9080"
```

Comment:
Is it required ? Why ?

Reply:
@sdake  Please chime in here, it was in the sample configmap you gave me.

Reply:
pilotEndpoint is required because the current validation code validates for it.  This value is unused in the codebase however.  I personally would like to remove all annotations that are unused to simplify things, but please don't overload PRs with multiple units of work.

I think it would make more sense to follow up this security change with a separate PR  that addresses configuration.

Reply:
SGTM.